docs/connectors/sinks/csv-sink.md

```diff
@@ -0,0 +1,58 @@
+# CSV Sink
+
```

Comment:
I think a "why use `sinks`" or "benefits of `sinks`" wouldn't hurt here (or maybe in the custom sinks README)?

Also, maybe known or potential limitations?

I'm thinking of cases where users maybe already made their own sink-like solution, or why they should (or shouldn't) make a new `sink` if one doesn't exist.

Reply:
A short explanation could be good. The main benefit that would need explaining is that Quix Streams provides durable compute/execution to ensure durability throughout distributed systems

Reply:
I'm gonna make a note for myself and update this part later.
I agree that it's worth telling more details about sinks overall.